data/json/mutations.json

```diff
@@ -662,7 +662,7 @@
         "type" : "mutation",
         "id" : "LACTOSE",
         "name" : "Lactose Intolerance",
-        "points" : -1,
+        "points" : -2,
```

Comment:
> Add small amounts of it to veggies and wheat and quite a bit to bone comestibles (bone broth, aspic).

The latter is a good example of a corner case whereby `vitamins` should be a member of `COMESTIBLE` to allow overriding the default values. I'd like to keep calcium rare however for balance but bones would provide for an interesting alternative source.